stories/RadioButton.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import { boolean, text, withKnobs } from '@storybook/addon-knobs'; // eslint-disable-line import/no-extraneous-dependencies
+import RadioButton from '../src/components/RadioButton';
+
+storiesOf('RadioButton', module)
+    .addDecorator(withKnobs)
+    .add('RadioButton', () => (
+        <div>
+            <RadioButton
+                disabled={boolean('Disabled 1', false)}
+                id={text('Id 1', 'radio-1')}
```

Comment:
ID's probably don't need knobs (it's sufficient to see them in the code example, don't you think?)

Reply:
removed it